modules/network-observability-architecture.adoc

```diff
@@ -0,0 +1,14 @@
+//Module included in the following assemblies:
+//
+// network_observability/understanding-network-observability.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-architecture_{context}"]
+= Network Observablity Operator architecture
+image::network-observability-architecture.png[Network Observability eBPF export architecture]
+The Network Observability Operator provides the `FlowCollector` API, which is instantiated at installation and configured to reconcile the `eBPF agent`, the `flowlogs-pipeline`, and the `netobserv-plugin` components. Only a single `FlowCollector` per cluster is supported.
+
+The `eBPF agent` runs on each cluster node in privileged mode to collect network flows. The `flowlogs-pipeline` receives the network flows data and enriches it with Kubernetes identifiers. The `flowlogs-pipeline` sends flow logs to Loki for storing and indexing. The `netobserv-plugin`, which is a dynamic {product-title} web console plugin, queries Loki to fetch network flows data. Cluster-admins can view the the data in the web console.
```

Comment:
> The `eBPF agent` runs on each cluster node in privileged mode to collect network flows.

Rather than "in privileged mode", could we say something like "with some privileges" ? The reason is that "privileged mode" refers to a specific mode and we actually try our best to avoid them running in that mode (by default, they don't) - but, for sure, the agent still have a bunch of privileges in any case